clusters/app.ci/hypershift-toolkit/rbac.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
-  name: hypershift-admin
+  name: hypershift-team
```

Comment:
This change has to be aligned with 

https://github.com/openshift/release/blob/0ff38126443dd0df982af89264081a13c31d401d/core-services/sync-rover-groups/_config.yaml#L20-L23

Reply:
ahh, so we need `hypershift-pool-admins` will switch

Reply:
Could you please also add `app.ci` into the `clusters` list in the config file?

Reply:
done